### DISCUSSION TITLE
書誌情報～予算の費目設定について教えて下さい

### DISCUSSION THREAD

**Discussion by Fumiko-Nishiura** [next-l/enju_leaf#1545] [Q&A] [answered]
書誌の詳細に、資料がどの予算から購入されたかを入力したいです。
学校図書館では、費目は３種類ぐらいです。
本校では、県費、寄贈、その他から選択できるようにしたいです。

マニュアルのどこを参照すればよいのでしょうか？

**Comment by masao** [comment, marked as answer]
https://next-l.github.io/manual/1.3/enju_setup_3.html#section3-9 初期設定マニュアルの3章にある予算種別をいくつか登録してみていただけると、所蔵情報の登録や編集の際に、予算情報を入力できるようになります。

**Reply by Fumiko-Nishiura** [reply]
第６章ばかり見ていました。
第３章を見て、予算種別は解決しました。
検索エンジンの設定をしてみると、噂どおりNDLサーチ等で検索できたので
めちゃくちゃ嬉しかったです。

明日また他の設定もしてみます。ありがとうございました。
